### ISSUE TITLE
Loading Crash with AP [1.12.2-11.13.0.40] toward Twilight Forest

### ISSUE THREAD

**Issue by Iteries** [sokratis12GR/ArmorPlus#200]
**ArmorPlus Version**: latest 1.12.2-11.13.0.40

**TheDragonLib Version**:   1.12.2-5.2.0

**MinecraftForge Version**: 1.12.2-14.23.2.2654

**Minecraft Version**: 1.12.2

**(Optional) Just Enough Items (JEI) Version**:  1.12.2-4.8.5.159

**(Optional) Tinkers' Construct Version**: n/a

**(Optional) CraftTweaker (CT) Version**: n/a

**Client or Server?**:
- [X] Client
- [ ] Server

**Crash Report (via. [Pastebin](http://pastebin.com/))**:  https://github.com/Iteries/Armorplus-DragonTeam-Log-Issue-Crash---Troubleshooting

( I don't have an account with pastebin but link provided should work.  Can't miss the title for the readme

**What happens / happened?**: I updated to latest armorplus and removed the old.  Went back into game and as it was going from late save point (happens to be in Twilight Forest)  Crashed on load.  Attempted again and crashed again.   I checked the logs and surprised it was armorplus.   I even remove the config and redid all the changes I have it set for as a new config.  Same thing happened.  So let me know if I screwed up somewhere in updating armorplus.

**How can I reproduce it?**:  Um, play in Twilight Forest as an older armorplus .38 > log out > then update to .40 then try to log back in.   Possible worlds collide 

**(Optional) Mod List**:
Mod list on Crash Report.    
All texture packs are by: http://bdcraft.net/community/purebdcraft/

**Comment by sokratis12GR** [comment]
Hello, thanks for your opening an issue ticket!
The crash seems to be because for some kind of a material the add effect don't match the add effect's level and duration
Could your provide me the config `registry.cfg` inside `config\armorplus`. But considering the amount of this type of issues happening lately I'll also investigate to see if the default config is messed up. But meanwhile be sure to delete the `registry.cfg` after you have provided it to me and test again.

**Comment by Iteries** [comment]
I deleted the armorplus folder again and instead of re-balancing the registry.cfg to fit with other mods prior to loading/playing the game I did it in the game one by one to see which is faulty line.  Sure enough it didn't like one change I had for emerald armor.  Just one level too high and I didn't realize it.   I added luck into emerald armor  with level 1 and it should have been 0.  My bad.

Thank you for the help.  Sometimes guess we need to double check things or a reminder of the basics.  

Conclusion: solved issue!
